README.md

```diff
@@ -241,6 +241,7 @@ Compatibility between this plugin and Zephyr Enterprise is:
 
 |Plugin|Zephyr Enterprise|
 |------|-----------------|
+|v2.4|v7.0 or later|
```

Comment:
do we have any new changes in this plugin, why are we releasing a new version ??

Reply:
There are parser changes